content/300-accelerate/350-static-ip.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Static IP"
+metaTitle: "Enable Static IP for Prisma Accelerate"
+metaDescription: "Learn enabling Static IP for Prisma Accelerate."
+---
+
+You can enable static IP for Accelerate when your security setup requires IP allowlisting or if you're implementing firewalls that only permit access from trusted IPs, ensuring controlled and secure database connections.
+
+![](./images/result-of-adding-static-ip-to-accelerate.png)
+
+:::info
+
+To enable static IP support for Accelerate within an existing or a new project environment, your workspace will need to be on our Pro or Business plans. Take a look at the [pricing page](https://www.prisma.io/pricing#accelerate) for more information.
+
+:::
+
+## Enable static IP in Accelerate
+
+You can opt-in to use static IP for Accelerate in the Platform Console in two ways:
+
+1. When enabling Accelerate for your project environment, opt-in to static IP after specifying your database connection string and connection pool region by toggling the **Static IP** switch in the **Network restrictions** section.
+
+2. For projects already using Accelerate, enable static IP by navigating to Accelerate settings and toggle the **Static IP** switch in the **Network restrictions** section.
```

Comment:
Can we add links to the console here somehow? Or some how show the UI? I just feel explaining this isn't as clear. Maybe break it down step by step rather than a sentence?

Reply:
@mhessdev see if it's more clear now 👁️